src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,33 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
+        # (hint, can do in 3 loc)
+        for x in range(i, len(arr)):
+            if arr[x] < arr[smallest_index]:
+                (arr[smallest_index], arr[x]) = (arr[x], arr[smallest_index])
```

Comment:
ahh, very nice, good to see you can just use this as a swapping method, rather than having to write out a swapping method. great work!